CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.3.0]
```

Comment:
I think this is worth a major version bump since the SDK might be our most widely used package and this is a change that may break people's workflows.
```suggestion
## [7.0.0]
```